extensions/wikia/AdEngine/js/video/player/porvata/porvata.js

```diff
@@ -73,6 +76,14 @@ define('ext.wikia.adEngine.video.player.porvata', [
 					video.ima.getAdsManager().dispatchEvent('wikiaAdPause');
 				});
 
+				if (params.autoPlay) {
+					video.addEventListener('wikiaAdStarted', function () {
+						if (isFirstPlay) {
+							video.mute();
+						}
+					});
+				}
+
```

Comment:
what about moving this block to a separate function? sth along `muteFirstPlay` or sth like this :D